arroyo/processing/processor.py

```diff
@@ -122,6 +123,18 @@ def __throttled_record(self) -> None:
         if time.time() - self.__last_record_time > METRICS_FREQUENCY_SEC:
             self.flush()
 
+    def record_timing(self, metric: MetricName, value: Union[int, float], additional_tags: Optional[Tags] = None) -> None:
```

Comment:
those wrapper functions don't seem to serve a purpose and don't add additional tags